forwarder/vppagent/conf/supervisord/telemetry.conf

```diff
@@ -0,0 +1,9 @@
+# Interval of data read procedure from the vpp in nanoseconds. Default value is 30 seconds.
+polling-interval: 30000000000
```

Comment:
Durations can be defined as string, implemented in: https://github.com/ligato/cn-infra/pull/408